tests/api/runtimes/test_spark.py

```diff
@@ -252,3 +253,55 @@ def test_cores(
 
         self.execute_function(runtime)
         self._assert_custom_object_creation_config(expected_cores=expected_cores)
+
+    @pytest.mark.parametrize(
+        "mount_v3io_to_executor",
+        [True, False],
+    )
+    def test_with_igz_spark(
+        self,
+        mount_v3io_to_executor,
+        db: sqlalchemy.orm.Session,
+        client: fastapi.testclient.TestClient,
+    ):
+        runtime = self._generate_runtime()
+
+        orig = os.getenv("V3IO_USERNAME")
+        os.environ["V3IO_USERNAME"] = "me"
+        try:
+            runtime.with_igz_spark(mount_v3io_to_executor=mount_v3io_to_executor)
```

Comment:
Why do you wrap this with a `try`? Are there cases where it's valid to have an exception thrown in this flow? If not, then don't catch any exception and let the test fail here.

Reply:
There is no catch. It's there for the finally clause.